levelkdev/swapr-dapp
# Remove local version of usePrevious hook

# Summary

Fixes #1026

We have two versions of usePrevious, one from react-use and one in our code. I removed the local version of this hook.

  # To Test

Images should help to localize changes.

Tokens list:
![Selection_156](https://user-images.githubusercontent.com/100695408/178431507-9163f550-b4d5-4098-9e89-9e0790e17a80.png)

Wallet modal:
![Selection_157](https://user-images.githubusercontent.com/100695408/178431522-aa8011cb-0575-445d-90ae-d15aaf6465ce.png)


Changes are in the tokens list and in the wallet modal.

1. Click the tokens list and check if everything works like before, also check if there is no issue with performance and speed.
2. Click the wallet modal and check if everything works like before, also check if there is no issue with performance and speed.
